### ISSUE TITLE
Can't build the docker images after aspnetcore-build image update to sdk 2.1.101

### ISSUE THREAD

**Issue by christiansparre** [dotnet-architecture/eShopOnContainers#543]
So about a day ago the aspnetcore-build:2.0 image was updated to sdk version 2.1.101 and now the build of the docker images fail

```
Building webstatus
Step 1/15 : FROM microsoft/aspnetcore:2.0.3 AS base
2.0.3: Pulling from microsoft/aspnetcore
723254a2c089: Pull complete
03ab0a5d7c3d: Pull complete
6dbb3614dd3b: Pull complete
02eb4fb302df: Pull complete
129bd50ee288: Pull complete
Digest: sha256:15ba4ece71c4219b2485851dc09dc80fb463ea68d053bd78e78553ea40385e48
Status: Downloaded newer image for microsoft/aspnetcore:2.0.3
 ---> 7f83f41eee6d
Step 2/15 : WORKDIR /app
Removing intermediate container f0648e5578fc
 ---> b8465295da52
Step 3/15 : EXPOSE 80
 ---> Running in 82cbf3b250e3
Removing intermediate container 82cbf3b250e3
 ---> 405bef4d5cad

Step 4/15 : FROM microsoft/aspnetcore-build:2.0 AS build
2.0: Pulling from microsoft/aspnetcore-build
3e731ddb7fc9: Pull complete
47cafa6a79d0: Pull complete
79fcf5a213c7: Pull complete
68e99216b7ad: Pull complete
0a45436f096d: Pull complete
5f8483081c16: Pull complete
2a51e585b106: Pull complete
ae521dfafffa: Pull complete
dc6a7c4b8711: Pull complete
8564dbbc6b7c: Pull complete
Digest: sha256:565ebb615692f07afa181bf9db4891035c14a01030f48de24d01817a93fa4b67
Status: Downloaded newer image for microsoft/aspnetcore-build:2.0
 ---> 203e3169173f
Step 5/15 : WORKDIR /src
Removing intermediate container 1e716ff10bb5
 ---> 0b1c25d7a540
Step 6/15 : COPY . .
 ---> f630ac9ebf36
Step 7/15 : RUN dotnet restore -nowarn:msb3202,nu1503
 ---> Running in 27acd2dadba1
/usr/share/dotnet/sdk/2.1.101/NuGet.targets(229,5): error MSB3202: The project file "/src/docker-compose.dcproj" was not found. [/src/eShopOnContainers-ServicesAndWebApps.sln]
ERROR: Service 'webstatus' failed to build: The command '/bin/sh -c dotnet restore -nowarn:msb3202,nu1503' returned a non-zero code: 1
```

It seems that NuGet has started to treat MSB3202 as an error instead of a warning

Changing the build image to `aspnetcore-build:2.0.5-2.1.4` makes the build work again

It relates to this: https://github.com/NuGet/Home/issues/6651, where I experienced it trying to use the 2.1 preview 1 runtime

**Comment by CESARDELATORRE** [comment]
Thanks for the heads-up. I confirm I get the same issue.
I'm changing all the dockerfiles to aspnetcore-build:2.0.5-2.1.4 until this issue is solved.

I'll contact the Docker Tooling team and other ASP.NET folks.
Thanks, 👍

**Comment by mvelosop** [comment]
Closing this as it was fixed with commit b78daa46 on March 16, will switch back to aspnetcore-build:20 shortly.
